web/controllers/category_controller.ex

```diff
@@ -4,10 +4,11 @@ defmodule CodeCorps.CategoryController do
   alias CodeCorps.Category
   alias JaSerializer.Params
 
+  plug :load_resource, model: Category, only: [:show]
```

Comment:
I'm not sure I understand really the implicit behavior here. We need `load_resource` for all resources that aren't plugged through `load_and_authorize_resource`, is that it?